gen/gen.c

```diff
@@ -2941,10 +2952,44 @@ rfc2544_test(unsigned int n)
 		statistics_clear();
 		transmit_set(1, 1);
 
-		state = RFC2544_MEASURING0;
+		state = opt_rfc2544_warming_duration > 0 ? RFC2544_WARMING0 : RFC2544_MEASURING0;
+		break;
+
+	case RFC2544_INTERVAL0:
+		transmit_set(1, 0);
+		statistics_clear();
+		memcpy(&statetime, &currenttime_main, sizeof(struct timeval));
+		statetime.tv_sec += 1;	/* wait 2sec */
```

Comment:
Is this `+= 1` still required? I'm asking because [the below diff](https://github.com/iij/ipgen/pull/20/files#diff-36566fdf4d2fd4a4d6cec00eb5c1ac60424b1792fad7d46799e0d880b48210c4L3042) removed `+= 1` in favor of `+= opt_rfc2544_interval`.

Reply:
Indeed, I removed this in [66be869](https://github.com/iij/ipgen/pull/20/commits/66be869dc17c1889ceaeef50ce0b50e3e2343c67)